process_test.go

```diff
@@ -63,6 +63,28 @@ func TestProcessRunConcurrently(t *testing.T) {
 	}
 }
 
+func TestProcessRunWithArgs(t *testing.T) {
+	var done atomic.Bool
+	p := newConcurrentProcess(1)
+	echo := testSkipIfNoCommand(t, p, "echo hello")
```

Comment:
This test doesn't fail when `newCommandRunner` fails because it calls `t.Skip`.

Instead, we need to check if `echo` exists separately.

```go
if _, err := execabs.LookPath("echo"); err != nil {
    t.Skipf("echo command is necessary to run this test: %s", err)
}

c, err := p.newCommandRunner("echo hello", false)
if err != nil {
    t.Fatalf(`parsing "echo hello" failed: %v`, err)
}
```